assets.src/src/redmine-tracky/controllers/timer-controller.ts

```diff
@@ -52,17 +52,10 @@ export default class extends Controller {
   }
 
   private timeDiffToString(timeDiff: TimeDiff) {
-    const sign = timeDiff.minutes < 0 || timeDiff.seconds < 0 ? '-' : ''
-
-    return (
-      sign +
-      ['hours', 'minutes', 'seconds']
-        .map((v) => timeDiff[v as keyof TimeDiff])
-        .map((v) => Math.abs(Math.floor(v)))
-        .filter((v, i) => i !== 0 || v !== 0) // Remove hours if zero
-        .map((v) => v.toString().padStart(2, '0'))
-        .join(':')
-    )
+    const duration = Duration.fromObject(timeDiff)
+    const formattedDuration = duration.toFormat('hh:mm:ss').replaceAll('-', '')
```

Comment:
Verify ISO standard for negative timestamp

Reply:
According to the ISO_8601 standard, there seems to be a notation, where the offset is perfixed with a 'minus' sign to imply negative time:
https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC